server/lib/mail/templates/password-change.html

```diff
@@ -0,0 +1,63 @@
+<!DOCTYPE html>
```

Comment:
I don't see the equivalent of this in `mail-templates/`, did you forget to add it to this commit?

Reply:
I did indeed :)
It's been a while since I composed an email for something; seems like there's quite a bit of repetition ^^.